docs/sources/reference/api/docker_remote_api_v1.17.md

```diff
@@ -1068,6 +1068,10 @@ The archive may include any number of other files,
 which will be accessible in the build context (See the [*ADD build
 command*](/reference/builder/#dockerbuilder)).
 
+The build can be canceled by closing the HTTP request. This also means that
+if a client disconnects because it quit or was killed, the build will be
+canceled.
```

Comment:
Let's simplify/clarify and put the new information where the reader needs it: The build will also be canceled if the client drops the connection by quitting or being killed.